### ISSUE TITLE
Feature: Allow more than 2 portal pairs.

### ISSUE THREAD

**Issue by KaziPlays** [ChaosInitiative/Portal-2-Community-Edition#814]
### Which component should be improved?

Gameplay

### Describe your suggestion

Portal 2 can't render more than two portal pairs, allowing this would add some more puzzle oportunities.

**Comment by Luke18033** [comment]
Portal 2 is perfectly capable of rendering more than 2 portal pairs if you turn off `r_portal_fastpath`, which we've already done by default.

**Comment by Luke18033** [comment]
In what way does it "mess up world portals"?

**Comment by KaziPlays** [comment]
When you do that command it applies to the world portals as well, I think there should be separate commands.

**Comment by AWildErin** [comment]
Right, but he's asking *how* it messes with the world portals. Like what happens?

**Comment by Luke18033** [comment]
What benefit would that have?

**Comment by KaziPlays** [comment]
> Right, but he's asking _how_ it messes with the world portals. Like what happens?

Won't them overlap?

**Comment by KaziPlays** [comment]
It doesn't matter, you already said it's done so yeah

**Comment by luxeleios** [comment]
World portals might be sharing the same linkage IDs that your regular portals have, which could be the reason threy mess it up

**Comment by Luke18033** [comment]
...no, that has nothing to do with it, it's a rendering issue only. World portals don't even use the standard linkage ID system.
